build/resources/package/chocolatey/livestreamer-twitch-gui.nuspec.tpl

```diff
@@ -26,14 +26,14 @@ Example: `choco uninstall livestreamer-twitch-gui -y -params '"/Purge"'`
   [Chromium]: https://www.chromium.org/ "Chromium"
     </description>
     <projectUrl><%= homepage %></projectUrl>
-    <packageSourceUrl>https://github.com/bastimeyer/livestreamer-twitch-gui</packageSourceUrl>
-    <tags>livestreamer twitch gui streaming</tags>
+    <packageSourceUrl>https://github.com/streamlink/streamlink-twitch-gui</packageSourceUrl>
+    <tags>streamlink twitch gui livestreamer streaming</tags>
     <copyright>Sebastian Meyer</copyright>
-    <licenseUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/blob/master/LICENSE</licenseUrl>
-    <docsUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/wiki</docsUrl>
-    <bugTrackerUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/issues</bugTrackerUrl>
+    <licenseUrl>https://github.com/streamlink/streamlink-twitch-gui/blob/master/LICENSE</licenseUrl>
+    <docsUrl>https://github.com/streamlink/streamlink-twitch-gui/wiki</docsUrl>
+    <bugTrackerUrl>https://github.com/streamlink/streamlink-twitch-gui/issues</bugTrackerUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <iconUrl>https://cdn.rawgit.com/bastimeyer/livestreamer-twitch-gui/master/src/img/icon-256.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/streamlink/streamlink-twitch-gui/master/src/img/icon-256.png</iconUrl>
     <dependencies>
       <dependency id="livestreamer" version="1.12.2"/>
```

Comment:
I have force pushed the branch with some minor changes, in case you have already pulled it... 😇
When you're done with the `streamlink` package, just change the dependency here (or apply any other needed changes), send me a PR (based off of this branch), so that I know that it's ready. The `streamlink-twitch-gui` choco package can then be created (without any time pressure) once I've merged the PR, moved the repo and published `1.0.0`. Thanks!